libcloudforensics/providers/azure/internal/compute.py

```diff
@@ -275,3 +275,20 @@ def Delete(self) -> None:
     except CloudError as exception:
       raise RuntimeError('Could not delete snapshot {0:s}: {1:s}'
                          .format(self.resource_id, str(exception)))
+
+  def GrantAccessAndGetURI(self) -> str:
+    """Grant access to a snapshot and return its access URI.
+
+    Returns:
+      str: The access URI for the snapshot.
+    """
+    snapshot = self.az_account.compute_client.snapshots.grant_access(
```

Comment:
Is this really a `snapshot`? Or a `grant_access` operation?

Reply:
Ah, yeah, that's a pretty bad name. Updated to `access_request`